mParticle-Example/MPKitExample.m

```diff
@@ -30,10 +30,20 @@ - (MPKitExecStatus *)execStatus:(MPKitReturnCode)returnCode {
 
 #pragma mark Kit instance and lifecycle
 - (MPKitExecStatus *)didFinishLaunchingWithConfiguration:(NSDictionary *)configuration {
-    NSString *apiKey = configuration[@"<dictionary key to retrieve API Key>"];
-    if (!apiKey) {
+    NSString *consumerKey = configuration[PILGRIM_SDK_KEY];
+    NSString *secretKey = configuration[PILGRIM_SDK_SECRET];
+    BOOL persistLogs = configuration[PERSIST_LOGS];
```

Comment:
BOOLs are primitive in obj-c, you will need to also wrap this in a boolValue:
`[configuration[PERSIST_LOGS] boolValue]`